src/server.js

```diff
@@ -193,7 +230,8 @@ app.post("/render", checkSecret, async (req: $Request, res: $Response) => {
             );
             return await Promise.all(fetchPromises);
         } catch (err) {
-            handleFetchError(err, res);
+            const errorResponse = logAndGetError("FETCH FAIL", err, globals);
+            res.status(500).json(errorResponse);
```

Comment:
Why do we do a 500 for this instead of a 400 as with the others?

Reply:
That's what it did before. I wasn't going to change the error just yet. This diff is about getting better info.